tests/test_partnerships.py

```diff
@@ -48,15 +48,15 @@ def test_image_srcs_are_correct(self, mozwebqa):
             src = partnerships_page.image_source(image.get('locator'))
             if not image.get('img_name_contains') in src:
                 bad_images.append('%s does not contain %s' % (src, image.get('img_name_contains')))
-        Assert.equal(0, len(bad_images), '%s bad images found: ' % len(bad_images) + ', '.join(bad_images))
+        assert [] == bad_images
 
     @nondestructive
     def test_partner_form_is_visible(self, mozwebqa):
         partnerships_page = Partnerships(mozwebqa)
         partnerships_page.go_to_page()
         partner_form = partnerships_page.partner_form
-        Assert.true(partner_form.is_form_present)
-        Assert.true(partner_form.is_title_visible, 'The title is not visible on the form')
+        assert partner_form.is_form_present
```

Comment:
I know the old code didn't do it, but should we add a descriptive failure message? If this fails we'll just know that False != True